app/controllers/serviceLearning/routes.py

```diff
@@ -191,6 +210,55 @@ def renewCourse(courseID, termID):
 
     return "", 500
 
+@serviceLearning_bp.route('/serviceLearning/print/<courseID>', methods=['GET'])
+def printCourse(courseID):
+    """
+    This function will print a PDF of an SLC proposal.
+    """
+    instructors = CourseInstructor.select().where(CourseInstructor.course==courseID)
+    courseInstructors = [instructor.user for instructor in instructors]
+    theCreator = Course.select().join(User).where(Course.createdBy.username, Course.id==courseID)
+    if g.current_user.isCeltsAdmin or g.current_user in courseInstructors or g.current_user in theCreator:
```

Comment:
theCreator should be a list of users rather than courses. Or don't use a list check at all and use .exists() on the query